jazz_test-lambda/components/config.js

```diff
@@ -0,0 +1,49 @@
+// =========================================================================
+// Copyright � 2017 T-Mobile USA, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+
+/**
+Nodejs Template Project
+/**
+	Nodejs Template Project
+  @module: config.js
+  @description: Defines variables/functions to retrieve environment related data
+	@author:
+	@version: 1.0
+**/
+
+var getStageConfig = (event) => {
+  var stage;
+  
+  if (event && event.awslogs && event.awslogs.data) {

```

Comment:
Why is this required? Shouldn't this look like other config.js files - https://github.com/tmobile/jazz/blob/master/jazz_environments/components/config.js?

Reply:
This newer version of config tries to consolidate multiple versions (https://github.com/tmobile/jazz/blob/master/jazz_cloud-logs-streamer/components/config.js) while making it concise and (arguably) more performant.